tasks/discover-granules/tests/index.js

```diff
@@ -422,3 +428,45 @@ test.serial('checkGranuleHasNoDuplicate throws an error on an unexpected API lam
       granulesRevert();
     }
   });
+
+test('discover granules sets the GRANULES environment variable and logs the granules', async (t) => {
+    const { event } = t.context;
+
+    event.config.collection.provider_path = '/granules/fake_granules';
+    event.config.provider = {
+      id: 'MODAPS',
+      protocol: 'http',
+      host: '127.0.0.1',
+      port: 3030
+    };
+
+    t.falsy(process.env.GRANULES);
+
+    const loggerInfoSpy = sinon.spy(Logger.prototype, 'info');
+
+    try {
+      await validateConfig(t, event.config);
+
+      await discoverGranules(event);
+
+      t.truthy(process.env.GRANULES);
+
+      const granules = JSON.parse(process.env.GRANULES);
+      t.deepEqual(granules, [
+        'granule-1',
+        'granule-2',
+        'granule-3'
+      ]);
+
+      // Check that the second log.info() call had the granules set
+      const loggerCall = loggerInfoSpy.getCall(1);
+      t.deepEqual(loggerCall.thisValue.granules[
```

Comment:
It horrifies me that this is passing. There is a comma missing between `loggerCall.thisValue.granules` and `[`, which happens to all of us, but then ... why is it passing? Javascript is doing some awful interpretation of this line and coming up with ... I don't know what.

In any case, without that comma, it's definitely not testing what you think it's testing.